templates/individual_response/interstitial.html

```diff
@@ -17,6 +17,7 @@ <h1 class="u-mt-m">{{_("If you can't answer questions for others in your househo
         onsButton({
             "text": _("Request separate census"),
             "classes": 'btn u-mb-m u-mt-s btn--link',
+            "url": "",
```

Comment:
The chevron is missing from the button the as the `url` property is empty. For this to work we need to remove the `POST` from the endpoint and on a GET request determine the URL for the next page. That URL should be used here, and will be whatever we currently redirect to on the POST request.